rasa/cli/test.py

```diff
@@ -78,6 +78,8 @@ def test_core(args: argparse.Namespace) -> None:
         )
 
         if args.evaluate_model_directory:
+            if os.path.isfile(args.model):
```

Comment:
Can we please add a unit test for this?

Reply:
Adding a unit test is probably easier if you move these lines to `test_core_models_in_directory` .

Reply:
depends which function you want to test:
- if you test `test_core` it should go into `tests.cli.test_rasa_test`
- if you test `test_core_models_in_directory` then I'd create a module `tests/test_test.py` and put the test in there

Reply:
So I'd always aim to have the tests in a mirroring file in the `tests` folder. So a module `foo` should have a corresponding test module `test_foo`. A module `some_package/bar` should have a test module `some_package/test_bar`. We are not consistent at all in `rasa` at the moment, but that's what we should aim for